go/oasis-test-runner/oasis/oasis.go

```diff
@@ -734,10 +733,6 @@ func (net *Network) startOasisNode(
 	args = append(args, baseArgs...)
 	args = append(args, extraArgs.vec...)
 
-	if !strings.HasPrefix(node.Name, "sentry-") && !strings.HasPrefix(node.Name, "ias-proxy") && len(net.byzantine) == 0 {
```

Comment:
So i'm not sure if the `&& len(net.byzantine) == 0` flag was needed before, since in the new code i also enable cert rotation in byzantine tests and tests don't fail